services/main/Controllers/User.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using main.DTOs;
+using main.Middlewares;
+using System.Security.Claims;
+
+namespace main.Controllers;
+
+[ApiController]
+[Route("api/v1/user")]
+public class UserController : ControllerBase
+{
+    private readonly ILogger<UserController> logger;
+    private readonly Domains.UserMngmt.UserService userService;
+
+    public UserController(
+        ILogger<UserController> logger,
+        Domains.UserMngmt.UserService userService)
+    {
+        this.logger = logger;
+        this.userService = userService;
+    }
+
+    [Authorize]
+    [HttpPost("savePhoneNumber")]
+    public async Task<OutputResponse<bool?>> SavePhoneNumber([FromBody] PhoneNumber userPhoneNumber)
```

Comment:
```suggestion
    public async Task<OutputResponse<bool?>> SavePhoneNumber([FromBody] PhoneNumberInput userPhoneNumber)
```

We want to append the types with `Input` or `Output` so that we can easily differentiate two types when we see them.

Also, is this the structure? `{"phoneNumber": "string"}`? We'd want the frontend to submit an object as body =)

Reply:
Yes, it allows frontend to submit object as body.